src/main/java/com/unascribed/fabrication/mixin/c_tweaks/feather_falling_no_trample/MixinFarmBlock.java

```diff
@@ -0,0 +1,33 @@
+package com.unascribed.fabrication.mixin.c_tweaks.feather_falling_no_trample;
+
+import com.unascribed.fabrication.support.EligibleIf;
+import net.minecraft.block.Block;
+import net.minecraft.block.FarmlandBlock;
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.enchantment.Enchantments;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(FarmlandBlock.class)
+@EligibleIf(configEnabled="*.feather_falling_no_trample")
+public class MixinFarmBlock extends Block {
+
+    public MixinFarmBlock(Settings settings) {
+        super(settings);
+    }
+
+    @Inject(method="onLandedUpon", at=@At("HEAD"), cancellable = true)
+    public void onLandedUpon(World world, BlockPos pos, Entity entity, float distance, CallbackInfo ci) {
+        if (entity instanceof LivingEntity
```

Comment:
Need a MixinConfigPlugin.isEnabled check

Reply:
Other discussion resolved so I'll fix this myself. Don't like blocking PRs on minor/style issues.

Reply:
nice since i um don't know what you're asking me todo, guess ill see shortly